src/site/_includes/components/Img.js

```diff
@@ -21,7 +21,7 @@ const generateSrc = (src, params) => client.buildURL(src, params);
  * @return {string}
  */
 const Img = function (args) {
-  const {src, alt, width, height, className, linkTo} = args;
+  const {src, alt, width, height, className, linkTo, caption, figure} = args;
```

Comment:
Is the figure option supposed to allow you to pass figure modifier classes?

I have to be honest that I'm not super keen on it, mostly because I don't love the original CSS I wrote for figure. It feels like this sort of compounds that technical debt by baking it into this shortcode.

I do like the idea that if someone provides a `caption` that we wrap the whole thing in a figure/figcaption. That seems like a pretty simple convenience that would satisfy many use cases.

A few thoughts:

**Option 1**
If folks want to use a fancy figure class we tell them to write it all out. This is similar to what we instruct folks to do on d.c.c.

<figure>
  {% Img ... %}
  <figcaption>...</figcaption>
</figure>

https://developer.chrome.com/docs/handbook/components/#full-bleed-images

**Option 2**
Have a figure shortcode for fancier use cases like floating and going fullbleed

Some example:
```md
{% Img ...%}
<!-- outputs: -->
<img src=...>

{% Img caption="..." %}
<!-- outputs: -->
<figure>
  <img src="...>
  <figcaption>...</figcaption>
</figure>

{% Figure caption='hello world', className="w-figure--inline-right" %}
{% Img ... %}
{% endFigure %}
<!-- outputs: -->
<figure class="inline-right">
  <img src="...>
  <figcaption>...</figcaption>
</figure>
```

I think I'm preferential to option 1 because it means we overall do less work in this PR (less risk that we break something) and we don't overload the Img shortcode. But I realize that a lot of work has already gone into this and maybe doing something like option 2 would ultimately be easier? What do you think @MichaelSolati ?